opm/simulators/flow/ExtraConvergenceOutputThread.cpp

```diff
@@ -57,8 +57,13 @@ namespace {
             "ReportStep"s,
             "TimeStep"s,
             "Time"s,
-            "CnvErrPvFrac"s,
             "Iteration"s,
+            "CnvPvFracConv"s,
+            "CnvPvFracRelax"s,
+            "CnvPvFracUnconv"s,
+            "CnvCellCntConv"s,
+            "CnvCellCntRelax"s,
+            "CnvCellCntUnconv"s,
```

Comment:
This defines 6 columns, expecting the vectors in report.cnvPvSplit() to have exactly 3 members. I suggest adding a safety here: if there are fewer or more than expected, make some dummy column headers (or only dummies) to ensure the rest of the columns get the correct labels.

Alternatively, make the vectors arrays with three elements. But that would make it harder to extend to arbitrary histograms as hinted at in the title.